### PR TITLE
Solve multiple choice rule

### PR DESCRIPTION
**Pull request by mauriciosouto**

### CODE REVIEW

File: backbone/src/main/java/org/researchstack/backbone/model/navigable/StepNavigationClause.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import org.researchstack.backbone.utils.StepResultHelper;
 
 import java.io.Serializable;
+import java.lang.reflect.Array;
```
Comment (peterTarmac) [inline backbone/src/main/java/org/researchstack/backbone/model/navigable/StepNavigationClause.java]: is this needed???

Comment (mauriciosouto) [reply backbone/src/main/java/org/researchstack/backbone/model/navigable/StepNavigationClause.java]: Nop

Comment (peterTarmac) [reply backbone/src/main/java/org/researchstack/backbone/model/navigable/StepNavigationClause.java]: there's one on the other file too

**Comment by peterTarmac** [comment]
👍

**Pull request merged**